plugin/tick.vim

```diff
@@ -2,7 +2,140 @@
 "
 " Much of this plugin was copied from Fatih's wonderful https://github.com/fatih/vim-go/, so send him your thanks!
 
+" install necessary Go tools
+if exists("g:tick_loaded_install")
+    finish
+endif
+let g:tick_loaded_install = 1
+
+
+" these packages are used by vim-tick and can be automatically installed if
+" needed by the user with TickInstallBinaries
+let s:packages = [
+            \ "github.com/influxdata/kapacitor/tick/cmd/tickfmt"
+            \ ]
+
+" These commands are available on any filetypes
 command! TickFmt call tick#fmt#Format()
+command! TickInstallBinaries call s:TickInstallBinaries(-1)
+command! TickUpdateBinaries call s:TickInstallBinaries(1)
+command! -nargs=? -complete=dir GoPath call go#path#GoPath(<f-args>)
```

Comment:
@mkboudreau Nope, that is fine. I am not too familiar with VIMscript so I honestly didn't know. If all it requires is that we have `go` installed then that is fine, since as you point out it was already assumed to be available.